src/styles/card.scss

```diff
@@ -0,0 +1,88 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  width: 200px;
+  box-sizing: border-box;
```

Comment:
better to apply 'box-sizing: border-box;' to every elements, so you can move it to the `*{ box-sizing: border-box; }`

Reply:
you already added box-sizing, you can delete it here